### ISSUE TITLE
Password: feedback width not equals to parent input

### ISSUE THREAD

**Issue by pizzi80** [primefaces/primefaces#7065]
**Describe the defect**

the width of the feedback is not equal to the parent input

**Reproducer**

![Schermata da 2021-03-03 10-40-50](https://user-images.githubusercontent.com/17908214/109786837-e26ac800-7c0d-11eb-8e5d-6f7f35a910ab.png)

**Environment:**
 - PF Version: _10.0_
 - JSF + version:  _2.3_
 - Affected browsers: _Chrome, iOS, ALL_

**To Reproduce**
1. Go to [https://www.primefaces.org/showcase/ui/input/password.xhtml?jfwid=88a94](url)
2. Click on Password Feedback example
3. Type 'a'

**Expected behavior**
The feedback should have the same width of the parent input
like the PrimeVue Password component

![Schermata da 2021-03-03 10-41-16](https://user-images.githubusercontent.com/17908214/109787650-bc91f300-7c0e-11eb-8da7-0046dce835cc.png)

**Example XHTML**
```html
<h:form id="frmTest">
  <h5>Feedback</h5>
  <p:password id="feedback" value="#{passwordView.password}" feedback="true"/>
</h:form>
```

**Example Bean**
```java
@Named
@RequestScoped
public class PasswordView implements Serializable {

    private String password;
    public String getPassword()  { return password; }
    public void setPassword(String password1) { this.password = password; }

}
```

**Comment by melloware** [comment]
Inline is even more wrong...
![image](https://user-images.githubusercontent.com/4399574/109873248-80c15280-7c3b-11eb-9d3c-d5a15a1b1a13.png)

**Comment by melloware** [comment]
@tandraschko I think we should assign to @mertsincan .  This is a regression in behavior from 8.0 when Mert added all the new animation and overlay stuff.  So I think this is a broken regression that should be fixed before 10.0.

**Comment by tandraschko** [comment]
for me its not a vital bug but please ask him.
